cache/class-vary-cache.php

```diff
@@ -585,9 +585,9 @@ private static function send_vary_headers() {
 			$sent_vary = true;
 
 			if ( self::is_encryption_enabled() ) {
-				header( 'Vary: X-VIP-Go-Auth' );
+				header( 'Vary: X-VIP-Go-Auth', false );
 			} else {
-				header( 'Vary: X-VIP-Go-Segmentation' );
+				header( 'Vary: X-VIP-Go-Segmentation', false );
 			}
 
 			if ( defined( 'WP_DEBUG' ) && true === WP_DEBUG ) {
```

Comment:
Probably want to add the `false` to the header() call here too.

Reply:
Not strictly necessary (since it's unlikely that someone else would send the debug header), but wouldn't hurt.